tcx-proto/src/sui.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+package transaction;
+
+message SuiTxInput {
+    string intent = 1;
+    string tx_data = 2;
```

Comment:
SuiTxInput should be generalized, including from, to, value, data and other information. The current user does not know how to construct tx_data